wrdcld/__init__.py

```diff
@@ -26,7 +28,10 @@ def make_word_cloud(
     word_padding: int = 0,  # TODO
     scaling_func: Callable[[float], float] = math.sqrt,
     mask=None,  # TODO
+    seed: int | float | str | bytes | bytearray | None = None,
 ):
+    if seed is not None:
```

Comment:
ℹ️ Suggestion: We should not set the global seed, but we can keep our own local `Random` instance. That way, people can get determinism from wrdcld without upsetting the randomness in the rest of their script (whatever it may be).

See: https://stackoverflow.com/a/37356024

Reply:
but we don't touch the random state unless a seed is supplied, no? If the user is controlling it globally then they just wouldn't touch this?

Reply:
The point is we _do_ touch the (global) random state. If the user wanted to do that, they could just call random.seed themselves before calling make_word_cloud. If we instantiate our own random generator and only seed that one, then this argument _only_ affects randomness within wrdcld, and nothing else in a user's library or script. They may want "true" randomness elsewhere but a handy way to get determinism from wrdcld.

(But we can do this in a separate PR for sure)